### PR TITLE
[FEAT] account tracing context and allow tracing for exports

### PR DESCRIPTION
**Pull request by aricart**

### CODE REVIEW

**Review by kozlovic — CHANGES_REQUESTED** [review]
You are missing the stream import's "allow trace" boolean.

**Review by kozlovic — CHANGES_REQUESTED** [review]
You are missing the stream import's "allow trace" boolean.

File: tests/imports_test.go [hunk 1]
```diff
@@ -358,6 +361,23 @@ func (t *ProviderSuite) Test_NewStreamImportSkToken() {
 	t.NoError(si.SetToken(token))
 }
 
+func (t *ProviderSuite) Test_NewStreamImportAllowTracing() {
```
Comment (kozlovic) [inline tests/imports_test.go:364]: nit: No reason to have `New` in the test name? (my guess is copy/paste from previous test that had it).

**Review by kozlovic — CHANGES_REQUESTED** [review]
Would rename a test, but otherwise LGTM.

File: tests/imports_test.go (hunk 1 above)
Comment (aricart) [reply tests/imports_test.go]: fixed

**Pull request merged**